ckan/setup/GSA_prerun.py

```diff
@@ -50,5 +50,6 @@ def check_solr_connection(retry=None):
         pr.update_plugins()
         pr.check_datastore_db_connection()
         pr.init_datastore_db()
-        check_solr_connection()
+        # This function does not work, but solr is up
+        # check_solr_connection()
```

Comment:
I know we had to overwrite the upstream version of this function when we moved to solr8 and/or solr cloud, maybe it would work now?

Reply:
Nope, this is cloud.gov deployment. As long as all the pieces are there (db, solr, etc) then this should just work.

Reply:
Sweet. Will give this a shot then.

Reply:
Added solr back into GSA_prerun.py, removed some bash for solr setup.